### ISSUE TITLE
upgrade to 2.x API

### ISSUE THREAD

**Issue by bdupras** [oesmith/ralio#32]
Upgrade the Rally API calls to 2.x.  This requires removing the use of adhoc.js since it's not supported in 2.x.

**Comment by igorescobar** [comment]
Yeah, but I think it would be better if we wait until it become stable, right?
